lib/Conversion/TorchToLinalg/Pooling.cpp

```diff
@@ -72,12 +72,13 @@ checkAndGetPoolingParameters(OpTy op, ConversionPatternRewriter &rewriter,
   return success();
 }
 
+
 // Creates a pooling operation based on the type specified by `OpTy` and
 // arguments passed.
 template <typename OpTy>
 static LogicalResult createPoolingOp(
     Operation *op, ConversionPatternRewriter &rewriter, Value self,
-    bool supportNonFPInput, bool ceilMode,
+    bool supportNonFPInput, bool ceilMode, int64_t dimension,
```

Comment:
nit: I would call this `dimensionality` to avoid confusion with the use of `dimension` to denote a specific axis of a tensor

Reply:
Fixed